### PR TITLE
Automated Tests with Azure blob storage using Azurite

### PR DESCRIPTION
**Pull request by ikivanc**
This is a technology specific automated test sample for Azure blob storage using Azurite.
This document covers how to run automated tests on local and pipeline.

### CODE REVIEW

**Review by katyamust — APPROVED** [review]

File: automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# Using Azurite to run blob storage tests in pipeline
+
+This document determines the approach for writing automated tests with a short feedback loop (i.e. unit tests) against security considerations (private endpoints) for the Azure Blob Storage functionality.
+
+Once private endpoints are enabled for the Azure Storage accounts, the current tests will fail when executed locally or as part of a pipeline because this connection will be blocked.
+
+## Utilize an Azure Storage emulator - Azurite
+
+To have a local Azure Blob Storage, there is [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator). The Storage Emulator currently runs only on Windows. If you need a Storage Emulator for Linux, one option is the community maintained, open-source Storage Emulator [Azurite](https://github.com/azure/azurite).
```
Comment (TessFerrandez) [inline automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md:9]:
```suggestion
To emulate a local Azure Blob Storage, we can use [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator). The Storage Emulator currently runs only on Windows. If you need a Storage Emulator for Linux, one option is the community maintained, open-source Storage Emulator [Azurite](https://github.com/azure/azurite).
```

File: automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+   python -m pytest ./tests
+   ```
+
+After running tests, you can see the files in your local blob storage
+
+![https local blob](images/http_local_blob_storage.png)
+
+## 3. Run tests on Azure Pipelines
+
+After running tests in our local we need to make sure these tests pass on Azure Pipelines too. We have 2 options here, we can use docker image as hosted agent on Azure or using npm package in the Pipeline steps.
```
Comment (TessFerrandez) [inline automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md:141]:
```suggestion
After running tests locally we need to make sure these tests pass on Azure Pipelines too. We have 2 options here, we can use docker image as hosted agent on Azure or install an npm package in the Pipeline steps.
```

**Review by TessFerrandez — APPROVED** [review]
A few nits but otherwise LGTM

File: automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md (hunk 2 above)
Comment (ikivanc) [reply automated-testing/tech-specific-samples/blobstorage-unit-tests/readme.md:141]: Thanks Tess these are fixed

**Pull request merged**